packages/service/src/domain/commands/processors/skip-command.ts

```diff
@@ -2,9 +2,12 @@ import CommandDefinition from '../model/command-definition';
 import CommandProcessingResponse, { makeSingleTextProcessingResponse } from '../model/command-processing-response';
 import * as EventStreamService from '../../../event-stream/event-stream-service';
 import { SkipEvent } from '../../../event-stream/model/events';
+import Command from '../model/command';
 
-async function skipCommandProcessor() : Promise<CommandProcessingResponse> {
-  const event: SkipEvent = { id: 'SkipEvent' };
+async function skipCommandProcessor(command: Command) : Promise<CommandProcessingResponse> {
+  const all = command.rawArgs === 'all';
+  const count = parseInt(command.rawArgs, 10);
```

Comment:
nie masz obsługi błędów na parsowaniu numbera, co jak ktoś poda jakiegoś innego stringa?
zobacz sobie jak jest random zaimplementowany dla inspiracji

Reply:
Sparsuje się do nulla w jsonie czego efektem na klientach będzie skipnięcie domyślnej 1.
Czy chcemy żeby to zachowanie wynikało wprost z kodu i sprawdzamy NaNa ?

Reply:
Zobacz jak jest w randomie, tam sprawdzamy NaN i leci wyjątek i tutaj bym tak samo zrobił

Reply:
Done, chociaż nie jestem przekonany czy ładnie.